extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -29,6 +29,25 @@ const TAG = 'amp-social-share';
 
 class AmpSocialShare extends AMP.BaseElement {
 
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+    /** @private {string} */
+    this.shareEndpoint_ = '';
+
+    /** @private {!Object} */
+    this.params_ = {};
+
+    /** @private {?../../../src/service/platform-impl.Platform} */
+    this.platform_ = null;
+
+    /** @private {string} */
+    this.href_ = '';
```

Comment:
better to have it as a nullable string and check if this is a string using dev().assertString() where needed?